CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ accidentally triggering the load of a previous DB version.**
 * #5584 Reduce decompression during constraint checking
 * #5530 Optimize compressed chunk resorting
 * #5639 Support sending telemetry event reports
+* #5586 Index scan support during UPDATE/DELETE
```

Comment:
i think you need to rebase this and adjust how changelog entries are handled

Reply:
Fixed.